test/jdk/java/lang/Class/forName/NonJavaNames.java

```diff
@@ -44,9 +57,50 @@ public Baz2(){}
         return new Baz2();
      }
 
-    public static void main(String[] args) throws Exception {
-        NonJavaNames.Baz bz = new NonJavaNames.Baz();
+    private static final String SRC_DIR = System.getProperty("test.src");
+    private static final String TEST_SRC = SRC_DIR + "/classes/";
+    private static final String TEST_CLASSES = System.getProperty("test.classes", ".");
+    Path dhyphenPath, dcommaPath, dperiodPath, dleftsquarePath, drightsquarePath, dplusPath, dsemicolonPath, dzeroPath, dthreePath, dzadePath;
```

Comment:
These variables should belong to the `createInvalidNameClasses` method.  You can simply add `Path` type declaration in line 78-87.